sdk/src/main/scala/com/horizen/block/MainchainTransaction.scala

```diff
@@ -113,6 +119,7 @@ object MainchainTransaction {
       }
     }
 
-    new MainchainTransaction(transactionBytes.slice(offset, currentOffset), version, crosschainOutputsMap)
+    val thisMainchainTransactionBytes = transactionBytes.slice(offset, currentOffset)
+    new MainchainTransaction(thisMainchainTransactionBytes, version, sidechainCreationOutputsData, forwardTransferOutputs)
```

Comment:
can be as before `new MainchainTransaction(transactionBytes.slice(offset, currentOffset), version, sidechainCreationOutputsData, forwardTransferOutputs)`
`val thisMainchainTransactionBytes` can be removed